modules/installation-network-user-infra.adoc

```diff
@@ -62,6 +69,13 @@ You must configure the network connectivity between machines to allow cluster
 components to communicate. Each machine must be able to resolve the host names
 of all other machines in the cluster.
 
+ifdef::ibm-z-kvm[]
+[NOTE]
+====
+The {op-system-base} KVM host must be configured to use bridged networking in libvirt or MacVTap to connect the network to the virtual machines. The virtual machines must have access to the network, which is attached to the {op-system-base} KVM host. Virtual Networks (i.e. NAT) within KVM are not a supported configuration.
+====
```

Comment:
Generally we try to avoid using parenthetical statements. Would something along the lines of the following make sense?

Virtual Networks, like network address translation (NAT), within KVM are not a supported configuration.